cores/arduino/main.cpp

```diff
@@ -45,6 +45,9 @@ int main( void )
 
 	for (;;)
 	{
+		// Define by Ethernet library. It is defined as __weak.
+		stm32_eth_scheduler();
```

Comment:
This is the idle task for handle timer and data reception. It must be called as much as possible. 
I put it here to be sure it is called at least one time per loop.